src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -4025,29 +4025,40 @@ module Queries = struct
           | None ->
               []
         in
-        let txns =
+        let txns : Transaction_hash.User_command_with_valid_signature.t list =
           (* Transactions as identified by IDs.
              This is a little redundant, but it makes our API more
              consistent.
           *)
+          let with_valid_signature user_cmd =
```

Comment:
Can you inline this function? I’m scared that someone else may use this without us knowing and then bypassing the comment justification pattern match.

Reply:
Done.